src/footer.ejs

```diff
@@ -10,14 +10,7 @@
                 © Guardian News and Media Limited or its affiliated companies. All rights reserved.
             </div>
         </footer>
-        <script async>
-            /* global System */
-            System.import('js/app').catch(function(e) {
-                setTimeout(function() {
-                    throw e;
-                });
-            });
-        </script>
+        <script src="./js/bundle.js" async defer></script>
```

Comment:
We don't need to include `system.js` or `config.js` anymore if we're loading a SFX bundle?

Alternatively, we still include system+config and just do `jspm bundle --inject` (https://github.com/jspm/jspm-cli/blob/master/docs/production-workflows.md#loading-a-bundle-automatically-inject)